src/analog/constants/commandRelations.ts

```diff
@@ -3,16 +3,18 @@
 import * as downlinkCommands from '../commands/downlink/index.js';
 import * as uplinkCommands from '../commands/uplink/index.js';
 
+export const downlinkCommandById = Object.fromEntries(
```

Comment:
do we really need this change?

Reply:
downlinkCommandById in commandRelations.ts now